sdk/communication/azure-communication-chat/azure/communication/chat/aio/_chat_thread_client_async.py

```diff
@@ -242,6 +243,8 @@ async def send_typing_notification(
     ) -> None:
```

Comment:
If you wanted, you could use the `*` syntax:
```python
async def send_typing_notification(
        self,
        *,
        sender_display_name: Optional[str] = None,
        **kwargs
    ) -> None:
```

Reply:
Is this the prefer format in async methods?

Reply:
Usually, yes - it works better with intellisense.